### PR TITLE
Ddcnl 9032

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/connectors/MiddleConnector.scala [hunk 1]
```diff
@@ -35,6 +37,26 @@ class MiddleConnector @Inject() (http: HttpClient, httpHandler: HttpHandler)(imp
 
   private def url(path: String) = s"$serviceUrl$path"
 
+  def connectToAtsSaFields(
```
Comment (pascal22p) [inline app/connectors/MiddleConnector.scala:40]: don't mix testOnly and prod code. Use a new file.

**Review by pascal22p — CHANGES_REQUESTED** [review]

**Comment by pascal22p** [comment]
There are quite a bit of testOnly stuff. Maybe group them together like it has been done in TEN

**Comment by grantrobson** [comment]
> There are quite a bit of testOnly stuff. Maybe group them together like it has been done in TEN

Good point. I was wondering about a better way to do this. I'll follow the model in TEN :-)

File: app/connectors/MiddleConnector.scala (hunk 1 above)
Comment (grantrobson) [reply app/connectors/MiddleConnector.scala]: Done

**Review by pascal22p — APPROVED** [review]

**Pull request merged**